src/LinAlg/hiopMatrixRajaSparseTriplet.cpp

```diff
@@ -1074,17 +1099,17 @@ void hiopMatrixRajaSparseTriplet::set_Jac_FR(const hiopMatrixSparse& Jac_c,
 
   assert(nnz_ == nnz_Jac_c_new + nnz_Jac_d + 2*m_d);
   
-  if(J_c.row_starts_host == nullptr) {
-    J_c.row_starts_host = J_c.allocAndBuildRowStarts();
+  if(J_c.row_starts_ == nullptr) {
+    J_c.row_starts_ = J_c.allocAndBuildRowStarts();
   }
-  assert(J_c.row_starts_host);
-  int* Jc_row_st = J_c.row_starts_host->idx_start_;
+  assert(J_c.row_starts_);
+  int* Jc_row_st = J_c.row_starts_->idx_start_;
 
-  if(J_d.row_starts_host == nullptr) {
-    J_d.row_starts_host = J_d.allocAndBuildRowStarts();
+  if(J_d.row_starts_ == nullptr) {
+    J_d.row_starts_ = J_d.allocAndBuildRowStarts();
   }
-  assert(J_d.row_starts_host);
-  int* Jd_row_st = J_d.row_starts_host->idx_start_;
+  assert(J_d.row_starts_);
+  int* Jd_row_st = J_d.row_starts_->idx_start_;
 
```

Comment:
Should we use HiOp's `index_type` here and elsewhere in the code when accessing vector/matrix indices?

Reply:
Now I use `index_type` and `size_type` in class RowStartsInfo and the places to access these member variables.